apps/frontend/src/styles/pageStyles.tsx

```diff
@@ -10,27 +10,94 @@ export const useStyles = () => ({
     width: "100vw",
     height: "30%",
   },
+  HomePageBox: {
+    flexDirection: "column",
+    alignItems: "top",
+    justifyContent: "top",
+    width: "99.3vw",
```

Comment:
This is likely going to make the spacing funny or add a scroll bar on unusally sized screens. instead it should probably be set to 100vw, and the padding or margin should be removed elsewhere (setting `margin:0` and `padding:0` here might work)

Reply:
updated